cmd/update-rules/main.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/glog"
+	"gopkg.in/yaml.v2"
+	"k8s.io/publishing-bot/cmd/publishing-bot/config"
+)
+
+func main() {
+	// TODO: change to main once the default branch changes to main
+	mainBranch := "master"
+
+	branchExample := "-branch release-x.yy"
+	ruleExample := "-rule path/to/rule/file.yaml"
+	goVerExample := "-go 1.16.1"
+	outExample := "-o /tmp/rules.yaml"
+
+	branchUsage := "Branch of k/k to create rules for, for example: " + branchExample
```

Comment:
nit: since publishing-bot is used by projects other than kubernetes, we try not to introduce any k8s-specific terms in the code. would be good to remove "k/k" here :)